src/pipewire/pipewire.cc

```diff
@@ -103,8 +104,9 @@ class PipeWireOutput : public OutputPlugin
     int m_aud_format = 0;
     int m_core_init_seq = 0;
 
-    unsigned char * m_buffer = nullptr;
-    unsigned int m_buffer_at = 0;
+    timespec m_time {};
```

Comment:
`m_time` is not very descriptive. Can we use a better name, or at least add a comment indicating what this value means?

Reply:
Removed m_time in next version as using pw_stream_get_time_n for getting delay.